### ISSUE TITLE
Sample fails to compile: Cannot use t.busObj (type *TestBusObject) as type dbus.BusObject

### ISSUE THREAD

**Issue by pinpox** [soumya92/barista#160]
Hello, 
the example on how to create your own bar fails to build for me. I have take out the parts that involve the google API since I don't use gmail.

My code and the build.sh script is here: https://github.com/pinpox/pinpox-bar

Am I doing something wrong or is this an error?

```
Skipping GITHUB_CLIENT_ID, value not set
Skipping GITHUB_CLIENT_SECRET, value not set
Skipping OWM_API_KEY, value not set
# barista.run/base/watchers/dbus
../../../barista.run/base/watchers/dbus/testbus.go:152:10: cannot use t.busObj (type *TestBusObject) as type dbus.BusObject in return argument:
	*TestBusObject does not implement dbus.BusObject (missing StoreProperty method)
../../../barista.run/base/watchers/dbus/testbus.go:160:2: cannot use o (type *TestBusObject) as type dbus.BusObject in return argument:
	*TestBusObject does not implement dbus.BusObject (missing StoreProperty method)
```

**Comment by WGH-** [comment]
You might be missing `go.mod` in your project.

**Comment by soumya92** [comment]
Looks like go.mod hides breaking changes in dbus. Since a method was added, I think it should have been v6.

In any case, I've update the code and go.mod to point to the latest commit in dbus/v5 as of right now.

Fixed in 70b1053d314a3a38ceabe4d116670397744fec24.
